Document/0x06d-Testing-Data-Storage.md

```diff
@@ -333,6 +333,7 @@ To use Needle to read the Keychain, execute the following command:
 ```  
 
 ###### Validate the Data Protection Classes
+
 You can use FileDP module in order to verify Data protection class of the file or directory:
```

Comment:
```suggestion
You can use FileDP module in order to verify data protection class of the file or directory:
```

Reply:
Thanks Carlos. Let's wait for the answer and then change it accordingly.